tests/lint/unittest_lint.py

```diff
@@ -251,7 +251,7 @@ def reporter():
 @pytest.fixture
 def init_linter(linter: PyLinter) -> PyLinter:
```

Comment:
I don't like this fixture name, should be a noun like `initialized_linter` ? But it's outside of the scope of this MR.

Reply:
I'll do a follow-up.